docusaurus/docs/ui-extensions/register-an-extension.md

```diff
@@ -0,0 +1,128 @@
+---
+id: register-an-extension
+title: Register an extension from your app plugin
+sidebar_label: Register an extension
+sidebar_position: 1
+description: Learn how to register an extension from your app plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - links
+  - extensions
+  - app plugins
+---
+
+## Before you begin
+
+Be sure your plugin meets the following requirements before proceeding:
+
+- It must be an app plugin.
+- It must be preloaded by setting the [preload property](https://grafana.com/docs/grafana/latest/developers/plugins/metadata/#pluginjson) to `true` in the `plugin.json`.
+- It must be installed and *enabled*.
+
+## How to add an extension to Grafana core/plugins from your app plugin
```

Comment:
```suggestion
## How to extend the Grafana UI or an app plugin from your app plugin
```

Not sure we should have `/` in a section header. Open to other suggestions on title @josmperez